galactic/distribution.yaml

```diff
@@ -2988,6 +2988,15 @@ repositories:
       url: https://github.com/SteveMacenski/slam_toolbox.git
       version: galactic
     status: maintained
+  soccer_interfaces:
+    release:
```

Comment:
Consider adding a `source` entry to make it easier for people to find the source repository.

Reply:
Done.